tests/python/multidevice/test_deepseek_v3.py

```diff
@@ -62,12 +69,77 @@ def load_model(config: transformers.PretrainedConfig) -> transformers.PreTrained
     return transformers.AutoModel.from_config(config, trust_remote_code=True)
 
 
+class Executor(Enum):
+    # https://docs.pytorch.org/docs/stable/distributed.tensor.parallel.html
+    TORCH_TP = auto()
+    NVFUSER = auto()
+
+
+def parallelize_linear_with_nvfuser(
+    linear: torch.nn.Linear,
+    mesh: dist.device_mesh.DeviceMesh,
+    parallel_style: ParallelStyle,
+) -> torch.nn.Linear:
+    assert isinstance(linear, torch.nn.Linear), f"Unsupported layer: {linear}"
+
+    assert len(parallel_style.input_layouts) == 1, "Expect 1D mesh"
+    input_layout = parallel_style.input_layouts[0]
+
+    assert len(parallel_style.output_layouts) == 1, "Expect 1D mesh"
+    output_layout = parallel_style.output_layouts[0]
+
+    if isinstance(parallel_style, RowwiseParallel):
+        assert input_layout.is_shard(-1), "We only implemented TP."
+        assert output_layout.is_replicate(), "We only implemented TP."
```

Comment:
```Suggestion
        assert output_layout.is_replicate(), "We only implemented TP, which expects outputs to be replicated."

Reply:
Same for the other error messages.

Reply:
"which expects outputs to be replicated" doesn't add information `output_layout.is_replicate`.

Reply:
Printed output_layout to make the message more useful.